docs/admin/setup/distributed.rst

```diff
@@ -502,3 +502,29 @@ It relays message between instances:
 
    .. figure:: distributed/registry.png
       :align: center
+
+
+Running multiple instances locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+1. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``
+
+2. Create additional run configurations for each instance
+
+You will have to add CLI arguments to each additional run configuration to set the instance ID and the port,
+e.g. ``--server.port=8081 --eureka.instance.instanceId="Artemis:2"``. Also make sure that only one instance has
+the ``scheduling`` profile enabled:
+
+   .. figure:: distributed/run-config.png
+      :align: center
+
+3. Start the registry service, e.g., by running ``docker compose -f docker/broker-registry.yml up``
+
+4. Start the first instance with the default run configuration (no additional CLI arguments, ``scheduling`` enabled.)
+   and wait until it is registered in the registry
+
+5. Start the remaining instances
+
+You should now be able to see all instances in the registry interface.
```

Comment:
```suggestion
You should now be able to see all instances in the registry interface at ``http://localhost:????``.
```
Might be helpful to add the port here when running as described in step 3 via Docker Compose.